### PR TITLE
fix ocp builds

### PR DESCRIPTION
**Pull request by tchughesiv**

### CODE REVIEW

File: src/main/java/com/redhat/dbaasdemoapps/postgresql/application/PostgreSqlApplication.java [hunk 1]
```diff
@@ -7,6 +7,12 @@
 public class PostgreSqlApplication {
 
 	public static void main(String[] args) {
+		String property = "org.springframework.cloud.bindings.boot.enable";
+
+		String val = System.getProperty(property);
+		System.out.println("isOk " + val);
+		System.getProperties().setProperty(property, "true");
```
Comment (redhatHameed) [inline src/main/java/com/redhat/dbaasdemoapps/postgresql/application/PostgreSqlApplication.java:14]: Good idea, to make sure the properly enabled.

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #spring.datasource.driver-class-name=org.postgresql.Driver
-#spring.datasource.url=jdbc:postgresql://172.23.0.2:5432/upperio
-#spring.datasource.username=upperio_user
-##spring.datasource.password=upperio//s3cr37
+#spring.datasource.url=jdbc:postgresql://localhost:5432/test_db
+#spring.datasource.username=postgres
+#spring.datasource.password=postgres
```
Comment (redhatHameed) [inline src/main/resources/application.properties:4]: Is this something need to be reverted ?

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -12,5 +12,3 @@ spring.sql.init.mode=always
 
 #spring.data.mongodb.database=fruits
 #spring.data.mongodb.uri=mongodb://myuser:mypassword@localhost:27017
-org.springframework.cloud.bindings.boot.enable=false
-org.springframework.cloud.bindings.boot.postgresql.enable=false
```
Comment (redhatHameed) [inline src/main/resources/application.properties:16]: Any thoughts why this removed.

File: src/main/resources/application.properties (hunk 2 above)
Comment (tchughesiv) [reply src/main/resources/application.properties:4]: its commented out anyway, but i think generic is always better

File: src/main/resources/application.properties (hunk 3 above)
Comment (tchughesiv) [reply src/main/resources/application.properties:16]: no, i just copied the working code from the `ocp_build` branch. my guess though is that it was disabling the very class we needed for this to work w/ SBO

Comment (redhatHameed) [reply src/main/resources/application.properties:16]: I see, that make sense now, that why our deployment was not making connection.  thanks

**Review by redhatHameed — APPROVED** [review]

**Pull request merged**